<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amazone</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="amazoneSW">
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="loginFrame">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>120</number>
           </property>
           <property name="topMargin">
            <number>60</number>
           </property>
           <property name="rightMargin">
            <number>120</number>
           </property>
           <property name="bottomMargin">
            <number>60</number>
           </property>
           <item>
            <widget class="QFrame" name="innerFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>60</number>
              </property>
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="rightMargin">
               <number>60</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="loginLabel">
                <property name="font">
                 <font>
                  <pointsize>24</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Iniciar sesión</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="emailLabel">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Correo</string>
                </property>
                <property name="buddy">
                 <cstring>emailLE</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="emailLE">
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="passwordLabel">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Contraseña</string>
                </property>
                <property name="buddy">
                 <cstring>passwordLE</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="passwordLE">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="loginPB">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(120, 120, 255, 255));</string>
                </property>
                <property name="text">
                 <string>Iniciar Sesión</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="Line" name="loginLine">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="newLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>¿Eres nuevo en Amazone?</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="newUserPB">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Crear tu cuenta Amazone</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="newUserPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="newUserFrame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>120</number>
           </property>
           <property name="topMargin">
            <number>60</number>
           </property>
           <property name="rightMargin">
            <number>120</number>
           </property>
           <property name="bottomMargin">
            <number>60</number>
           </property>
           <item row="0" column="0">
            <widget class="QFrame" name="newInnerFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>60</number>
              </property>
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="rightMargin">
               <number>60</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="createLabel">
                <property name="font">
                 <font>
                  <pointsize>24</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Crear cuenta</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="newNameLabel">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Nombre</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="newNameLE">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="newMailLabel">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Correo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="newMailLE">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="newPasswordLabel">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Contraseña</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="newPasswordLE">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="createPB">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(120, 120, 255, 255));</string>
                </property>
                <property name="text">
                 <string>Crear tu cuenta Amazone</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="amazonePage">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="0" colspan="2">
         <widget class="QScrollArea" name="recommendationSA">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="recommendationWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>762</width>
             <height>198</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QScrollArea" name="auxScrollArea">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>762</width>
             <height>232</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="auxGrid">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinAndMaxSize</enum>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="searchLE">
          <property name="placeholderText">
           <string>Buscar...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="sortCB">
          <item>
           <property name="text">
            <string>Ordenar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Descendente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ascendente</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="departmentDB">
          <property name="currentText">
           <string>Todos los Departamentos</string>
          </property>
          <item>
           <property name="text">
            <string>Todos los Departamentos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alimentos y Bebidas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Libros</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Electrónicos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hogar y Cocina</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deportes y Aire Libre</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Lo más vendido:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
